build.sbt

```diff
@@ -9,7 +9,7 @@ lazy val scala3 = "3.3.0"
 ThisBuild / scalaVersion := scala3
 
 lazy val commonSettings = Seq(
-  version := "0.28.4",
```

Comment:
Could you rebase commit on a master branch?
There is no more need to update version by hand

Reply:
ok I did it